source/clear-linux/guides/maintenance/swupd-guide.rst

```diff
@@ -0,0 +1,160 @@
+.. _swupd-guide:
+
```

Comment:
After reviewing the 'update.rst' file, I think that your guide starts overshadowing it. 
https://clearlinux.org/documentation/clear-linux/guides/maintenance/update

In short, I think it would be best if you folded in the current content from `update.rst` into the current `swupd-guide`. I don't see any clear justification for `udpate.rst` to be a standalone document, given the more comprehensive guide you provide.